config/environments/production.rb

```diff
@@ -1,5 +1,7 @@
 # frozen_string_literal: true
 
+# require "active_support/core_ext/integer/time"
```

Comment:
Do we need this?

Reply:
Nope, good catch. That was a remnant from troubleshooting that I missed